src/formio/components/addressNL.ts

```diff
@@ -24,4 +34,5 @@ export interface AddressNLComponentSchema
   type: 'addressNL';
   deriveAddress: boolean;
   layout: 'singleColumn' | 'doubleColumn';
+  ofComponents?: AddressComponents;
```

Comment:
I tried using `components`, but ran into quite a lot of issues because of certain code operating on `components` (expecting it to be a fieldset/etc)

Reply:
hrm, unfortunate - was it formio.js code itself, or our code?

Either way - I'd rather see this as a subkey in `openForms` (see https://github.com/open-formulieren/types/blob/04925426f24a30887e5eee8202933404200896c2/src/formio/base.ts#L56) instead of another top-level key that pollutes the namespace - `openForms.components` could maybe be viable?

Reply:
It was our code and it may have been formio as well (though I don't remember it all that well). I'll change it to `openForms.components` :+1: